qualtran/bloqs/util_bloqs.py

```diff
@@ -50,8 +50,31 @@
     from qualtran.simulation.classical_sim import ClassicalValT
 
 
+class UtilBloq(Bloq, metaclass=abc.ABCMeta):
```

Comment:
I think this is reasonable, but please

 - Rename to `BookkeepingBloq`, which is more specific than "util" (a name I despise -- we can change the module later :). This also makes the properties make sense: a bookkeeping bloq is trivially controlled and does notn affect the t complexity
 - make private, i.e. `_BookkeepingBloq`

Reply:
renamed to `_BookkeepingBloq`